src/main/java/com/teammetallurgy/atum/blocks/AtumBlocks.java

```diff
@@ -119,8 +119,8 @@ public void registerBlocks() {
         GameRegistry.registerBlock(BLOCK_THINCRYSTALSTAINEDGLASS, ItemBlockStainedGlass.class, BLOCK_THINCRYSTALSTAINEDGLASS.getUnlocalizedName());
         GameRegistry.registerBlock(BLOCK_THINFRAMEDSTAINEDGLASS, ItemBlockStainedGlass.class, BLOCK_THINFRAMEDSTAINEDGLASS.getUnlocalizedName());
 
-        GameRegistry.registerBlock(BLOCK_SLABS, ItemBlockSlab.class, BLOCK_SLABS.getUnlocalizedName());
-        GameRegistry.registerBlock(BLOCK_DOUBLESLAB, ItemBlockSlab.class, BLOCK_DOUBLESLAB.getUnlocalizedName());
+        GameRegistry.registerBlock(BLOCK_SLABS, ItemBlockSlab.class, "slab");
+        GameRegistry.registerBlock(BLOCK_DOUBLESLAB, ItemBlockSlab.class, "doubleSlab");
```

Comment:
when you use getUnlocalizedName it returns tile.BlockName, you removed the "tile.", so the ID changes and removes blocks